authorize/grpc.go

```diff
@@ -56,6 +58,10 @@ func (a *Authorize) Check(ctx context.Context, in *envoy_service_auth_v3.CheckRe
 	if sessionState != nil {
 		s, err = a.getDataBrokerSessionOrServiceAccount(ctx, sessionState.ID, sessionState.DatabrokerRecordVersion)
 		if err != nil {
+			if status.Code(err) == codes.Unavailable {
```

Comment:
could we move this one level up and do the `if err != nil` as an `else if err != nil`